packages/contracts-bedrock/scripts/Config.sol

```diff
@@ -67,4 +87,44 @@ library Config {
     function drippieOwnerPrivateKey() internal view returns (uint256 _env) {
         _env = vm.envUint("DRIPPIE_OWNER_PRIVATE_KEY");
     }
+
+    /// @notice Returns the OutputMode for genesis allocs generation.
+    ///         It reads the mode from the environment variable OUTPUT_MODE.
+    ///         If it is unset, OutputMode.ALL is returned.
+    function outputMode() internal view returns (OutputMode) {
+        string memory modeStr = vm.envOr("OUTPUT_MODE", string("latest"));
+        bytes32 modeHash = keccak256(bytes(modeStr));
+        if (modeHash == keccak256(bytes("none"))) {
+            return OutputMode.NONE;
+        } else if (modeHash == keccak256(bytes("latest"))) {
+            return OutputMode.LATEST;
+        } else if (modeHash == keccak256(bytes("all"))) {
+            return OutputMode.ALL;
+        } else {
+            revert(string.concat("Config: unknown output mode: ", modeStr));
+        }
+    }
+
+    /// @notice Returns the latest fork to use for genesis allocs generation.
+    ///         It reads the fork from the environment variable FORK. If it is
+    ///         unset, NONE is returned.
+    ///         If set to the special value "latest", the latest fork is returned.
+    function fork() internal view returns (Fork) {
+        string memory forkStr = vm.envOr("FORK", string(""));
```

Comment:
I do think we can simplify by removing `Fork.NONE`. I also think it would be better to call it `Fork.BEDROCK`

Reply:
Thoughts on removing `Fork.NONE` for now and then we can reintroduce it in the future as `Fork.BEDROCK` when we standardize on the predeploy upgrade flow?

Reply:
I also wasn't super happy about adding the `Fork.NONE` option, but I need it to signal back from the `Config.fork()` function that the env var is not set, so I can then fallback to the deploy config one in the L2 genesis script.

Reply:
Ah ok, this makes sense, thoughts on just renaming it to `BEDROCK` for now?

Reply:
The default could be `BEDROCK` and we could remove the length check?

Reply:
Btw note that `NONE` is not Bedrock, but stands rather for _unspecified_.

Reply:
I need a way to signal from `Config` that the env var is not set. I didn't intend to support Bedrock, Regolith or Canyon, as we didn't before and I guess nobody needs it really.